Frameworks/include/CIImageInternal.h

```diff
@@ -17,14 +17,17 @@
 #pragma once
 
 #import <CoreImage/CIImage.h>
-#include "Starboard.h"
+#import <CoreGraphics/CGImage.h>
+#import <memory>
+#import <Starboard.h>
 
 @interface CIImage () {
-    idretain _cgImage;
+    woc::unique_cf<CGImageRef> _cgImage;
     idretain _color;
```

Comment:
>idretain [](start = 4, length = 8)

nit: Could change this to StrongId<> while we are at it.  (granted, it is a typedef)